.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Make a commit to Ichai5:master that deletes the file.  The PR will be automatically updated when the change is pushed to GitHub Ichai5:master.